arch/x86/core/ia32/crt0.S

```diff
@@ -41,7 +42,53 @@
 	GTEXT(_sys_resume_from_deep_sleep)
 #endif
 
+.macro install_page_tables
+#ifdef CONFIG_X86_MMU
+	/* Enable paging. If virtual memory is enabled, the instruction pointer
+	 * is currently at a physical address. There is an identity mapping
+	 * for all RAM, plus a virtual mapping of RAM starting at
+	 * CONFIG_KERNEL_VM_BASE using the same paging structures.
+	 *
+	 * Until we enable these page tables, only physical memory addresses
+	 * work.
+	 */
+	movl $Z_MEM_PHYS_ADDR(z_x86_kernel_ptables), %eax
```

Comment:
here comes the page table struct created by the python script right ?

Reply:
yes. `z_x86_kernel_ptables` is the product of gen_mmu.py.